packages/patternfly-4/react-table/src/components/Table/Body.tsx

```diff
@@ -37,6 +49,7 @@ class ContextBody extends React.Component<TableBodyProps, {}> {
           isInput: (event.target as HTMLElement).tagName !== 'INPUT',
           isButton: (event.target as HTMLElement).tagName !== 'BUTTON'
         };
+
         onRowClick(event, row, rowProps, computedData);
```

Comment:
Should we test if `onRowClick` exists first? That way you don't need to fire a stub function?

Reply:
That's what was in the previous version and it's not really that good. Firing empty function is nothig compared to checking if function exists. The undefined instead of empty function might hurt us in future when we forgot to check if this function is defined and blow UI.

Overall it boils down to be consistent and since almost every other component is using either `() => undefined`  or `Function()` instead of checking before call I think empty function is better.

Reply:
Yes, we use this pattern a lot with stateless components. Don't agree it would "blow UI", but don't feel strongly about it.